src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+  const basicCost = dailyCost * days;
+  const basicDiscount = 20;
+  const higherDiscount = 50;
+
+  let totalCost = basicCost;
+
+  if (days >= 3 && days < 7) {
+    totalCost = basicCost - basicDiscount;
```

Comment:
I don't think this is necessary. You could instead use return and place these variables in the return command instead. However, this works just as well.

Reply:
This is actually in the checklist point 3 :) try to use one line, then we don't have to create variable with `let`